content/spin/v2/triggers.md

```diff
@@ -13,9 +13,70 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/triggers.md"
 
 A Spin _trigger_ maps an event, such as an HTTP request or a Redis pub-sub message, to a component that handles that event.
 
-An application can contain multiple triggers, but they must all be the same type. For example, an application can contain triggers for multiple HTTP routes, or for multiple Redis pub-sub channels, but can't contain both an HTTP _and_ a Redis trigger.
+An application can contain multiple triggers. As of Spin 2.3 onward, a Spin application can use any combination of trigger types within a single application. For example, a single application can contain triggers for both HTTP routes and Redis pub-sub channels. 
+
+> Currently, only the HTTP template supports the `spin add` command. If you wish to use `spin add` to build an application with multiple triggers, **including a Redis trigger**, you should begin with the Redis trigger application. Then, `spin add` your HTTP trigger component(s) as demonstrated below. Please note that additional Redis trigger components cannot be added via `spin add` at this time.
+
+Here is a quick example of creating an application with both HTTP and Redis triggers:
+
+<!-- @nocpy -->
+
+```bash
+# Start with a Redis trigger application
+$ spin new -t redis-rust trigger-example
+Description: A multiple trigger example
+Redis address: redis://localhost:6379
+Redis channel: one
+# Change into to the application directory
+$ cd trigger-example 
+# Create HTTP trigger application
+$ spin add -t http-rust http-trigger-example  
+Description: A HTTP trigger example
+HTTP path: /...
+```
+
+The above `spin new` and `spin add` commands will automatically scaffold the following Spin manifest (`spin.toml` file):
```

Comment:
This is getting very long and intricate for the opening paragraphs of the page.  Consider moving this to a "Setting up Multiple Trigger Types" section further down, after the orientation material.

I'd also strongly suggest pruning the TOML down to what you want people to look at.  The current TOML block is about 30 lines, most of which are boilerplate like "authors" and "component.*.build"  or empty `allowed_outbound_hosts` lines.  These make it hard to the eye to home in on what's important or distinctive about this particular TOML file.

Reply:
Updated in new push. Thanks @itowlson!